SmartDeviceLink/SDLTextAndGraphicManager.m

```diff
@@ -439,6 +455,25 @@ - (SDLShow *)sdl_extractImageFromShow:(SDLShow *)show {
     return newShow;
 }
 
+- (nullable SDLShow *)sdl_extractUploadedImagesFromShow:(SDLShow *)show primaryGraphic:(nullable SDLArtwork *)primaryGraphic secondaryGraphic:(nullable SDLArtwork *)secondaryGraphic  {
```

Comment:
I'm a bit confused about the purpose of this method. The naming doesn't seem to be accurate though. It's checking if a passed graphic was uploaded and if it is, then using the previous show graphic? "Extract from" implies returning the graphics, which isn't what's happening here.

Reply:
Fixed. Renamed method